pkg/executor/push.go

```diff
@@ -128,9 +140,21 @@ func DoPush(image v1.Image, opts *config.KanikoOptions) error {
 		if err != nil {
 			return errors.Wrap(err, "getting tag for destination")
 		}
+		if opts.ImageNameDigestFile != "" {
```

Comment:
this implies `--digest-file` required for `--image-name-with-digest-file` ?
it should be fine to just use `--image-name-with-digest-file`

Reply:
Not sure I am following why line 143 implies --digest-file is required. The intent was to build up the list of images for each destination specified. Could you elaborate?